extensions/azurehybridtoolkit/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure SQL Hybrid Cloud Toolkit *(preview)*
 
-Adds a Jupyter Book that has several utilities for Azure SQL Hybrid Cloud.
+Adds a Jupyter Book that has several utilities for Azure SQL Hybrid Cloud. After installing the extension, open the Command Palette from the View menu typing a portion of the name of this toolkit. The Notebook should show in the Provided Notebooks panel of the ADS Notebooks Explorer.
```

Comment:
`Provided` should be `provided`

Reply:
I think capitalizing Provided is correct here - that's the full name of the view panel : 

![image](https://user-images.githubusercontent.com/28519865/112522653-20659280-8d5b-11eb-9bb2-811a9100ab4d.png)